From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Fri, 18 Aug 2023 11:01:51 +0200
Subject: [PATCH] HACK: ARM: i.MX8MP: skov: add temporary version

Add temporary machine version. Proper implementation of version detection
will implemented separately.

Signed-off-by: Oleksij Rempel <o.rempel@pengutronix.de>
---
 arch/arm/boards/skov-imx8mp/board.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boards/skov-imx8mp/board.c b/arch/arm/boards/skov-imx8mp/board.c
index 398c74677d46..657ac595084a 100644
--- a/arch/arm/boards/skov-imx8mp/board.c
+++ b/arch/arm/boards/skov-imx8mp/board.c
@@ -25,6 +25,12 @@ static int skov_imx8mp_probe(struct device *dev)
 	imx8m_bbu_internal_mmc_register_handler("SD", "/dev/mmc1.barebox", sd_bbu_flag);
 	imx8m_bbu_internal_mmcboot_register_handler("eMMC", "/dev/mmc2", emmc_bbu_flag);
 
+	/* TODO: until we have proper board version detection, we use a single
+	 * available board variant. At this point we don't have any other
+	 * variants, so this is fine. (2023.08.18)
+	 */
+	of_prepend_machine_compatible(NULL, "skov,imx8mp-skov-reva-mi1010ait-1cp1");
+
 	return 0;
 }
 
